packages/@aws-cdk/aws-lambda-python/lib/bundling.ts

```diff
@@ -87,7 +87,7 @@ export class Bundling implements CdkBundlingOptions {
     if (packaging.dependenciesFile) {
       bundlingCommands.push(`python -m pip install -r ${DependenciesFile.PIP} -t ${options.outputDir}`);
     }
-    bundlingCommands.push(`cp -R ${options.inputDir}/ ${options.outputDir}`);
+    bundlingCommands.push(`cp -a ${options.inputDir}/* ${options.outputDir}`);
```

Comment:
Including the `*` still drops hidden files. This needs to be removed.

```suggestion
    bundlingCommands.push(`cp -a ${options.inputDir}/ ${options.outputDir}`);
```

Reply:
Urm, I added a validation for it within the test, which passes.